src/urllib3/util/retry.py

```diff
@@ -269,6 +269,12 @@ def parse_retry_after(self, retry_after):
             retry_date_tuple = email.utils.parsedate_tz(retry_after)
             if retry_date_tuple is None:
                 raise InvalidHeader("Invalid Retry-After header: %s" % retry_after)
+            if retry_date_tuple[9] is None:  # Python 2
```

Comment:
Could we do this?

```python
if retry_date_tuple[9] is None:  # Python 2
    retry_date_tuple = retry_date_tuple[:8] + (0,) + retry_date_tuple[9:]
```

Also we should add a comment that Python 2 parsedate_tz() returns `None` for a timezone offset instead of 0 if no timezone is given.

Reply:
Done, I'll add the test and ping you :)